ROADMAP.md

```diff
@@ -6,44 +6,15 @@ AsyncAPI is a specification for defining message-driven APIs. We're on a mission
 ## How to Get Involved?
 AsyncAPI is an open-source community-driven project. We're happy to receive contributions from different people, no matter your skills or your area of expertise. There are many ways you can contribute to the project, not just code. There is always a need for: documentation, design, writing, evangelizing, code, issue triage, and more! We also welcome financial support and donations. Check out the [contributing guidelines](./CONTRIBUTING.md) to learn more.
 
-## Timeline
+## Versioning
+AsyncAPI schema is versioned according to [semver](https://semver.org/) guidelines. You can check latest [v2.0.0 here](https://asyncapi.io/docs/specifications/2.0.0).
 
-We're working hard to reach version 2.0.0, which will be full of great new features (and bug fixes.)
+### Release Candidates (RC)
```

Comment:
I'd not talk directly about RCs here. Maybe we should call them "prereleases" (as defined by Semver). We may also opt for `-alphaN` suffixes before we call them a "release candidate". RC should only be those versions we believe are done and therefore we want people to try and tell us if they're really done or not.

Also, not sure this is the right document for this information. Maybe we should have a separate VERSIONING.md document?

Reply:
Having separate RC and Alpha releases will just make it more confusing. To make it clear, RC doesn't guarantee anything. It's a candidate version for the upcoming release.

I would suggest we only follow one convention for now and not make it too complex atm. If in the future, we need to separate out we can.

I like the idea of explaining everything about RC in VERSIONING.md. Couldn't find the best place for it. Will create it now. 👍

Reply:
In some cases, we may find it useful to flag a spec version as alpha because we want to release different packages with the same version to see how everything fits together. We'll most probably do it to try things out separately, knowing it's far from being close to stable. Alpha is just another step in the software development cycle where you're experimenting, let's use it. A release candidate is a version that could potentially become the final version: https://en.wikipedia.org/wiki/Software_release_life_cycle.

For instance, I may want to release 2.1.0-alpha1 which **only** contains your MessageId proposal, so the team can start working on the tooling to support it. 2.1.0-alpha2 might contain a second feature we want to include, and so on. In these cases, we don't to communicate the users that we have a release candidate because we know it's far from being it. But that's ok, just because we release them it doesn't mean we have to communicate about them.